force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -228,18 +271,24 @@ public with sharing class ClassesObjectsMethods {
     public class Book {
         // Declare three public instance variables - title, author, pages
         // title
+        public string title;
         // author
+        public string author;
         // pages
+        public Integer pages;
     
         // Book constructor to initialize the title and author
-        public Book() {
+        public Book(string title, string author) {
             // Initialize the title and author
+            this.title = title;
+            this.author = author;
         }
     
         // Method that returns the details of the book
         public String getBookDetails() {
+            string bookDetails = 'Title: ' + title + ', ' + 'Author: ' + author;
```

Comment:
It's not required but general conversion to use this. for class variables inside of a class
```suggestion
            string bookDetails = 'Title: ' + this.title + ', ' + 'Author: ' + this.author;
```